### PR TITLE
{vis}[intel/2015b] VTK v6.3.0 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a183ceaa465f2f239d0b for a full test report.

File: easybuild/easyconfigs/v/VTK/VTK-6.3.0-intel-2015b-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+versionsuffix = '-Python-%s' % pyver
+
+mesaver = '11.0.8'
+dependencies = [
+    ('Python', pyver),
+    ('Mesa', mesaver, versionsuffix),
+    ('libGLU', '9.0.0', '-Mesa-%s' % mesaver),
+]
+
+configopts = "-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.so -DOPENGL_gl_LIBRARY=$EBROOTMESA/lib/libGL.so "
```
Comment (wpoely86) [inline easybuild/easyconfigs/v/VTK/VTK-6.3.0-intel-2015b-Python-2.7.11.eb]: `SHEXT`

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/57ecb9d8334f75a9c735 for a full test report.

**Pull request merged**